### PR TITLE
Ensured that 'portrait' is not the only orientation on Chrome for Android in manifest.json

### PR DESCRIPTION
**Pull request by euangoddard**
Supports null, 'portrait' and 'landscape'

### CODE REVIEW

**Comment by haydenbleasel** [comment]
Need to add the property to `data/defaults.json` and document it in the `readme.md`.

**Comment by euangoddard** [comment]
OK. I'll take a look at that when I get a chance. Do you see any merit in allowing overrides for all the of properties which are generated in manifest.json?

**Comment by haydenbleasel** [comment]
The RFG docs state the following for orientation:

> When present and display is standalone, force the screen to a particular orientation. Either `portrait` or `landscape`.

And the following for display:

> `browser` (the site is opened like any regular bookmark, in a new Chrome tab. Because Android Chrome does not actually support the browser attribute, the display attribute is completely absent of the generated manifest) or `standalone` (the web site is somehow treated like a native application. When the home screen link is clicked, a new Chrome instance is opened, the navigation UI is not displayed and the web site is displayed full-screen).

I'm not sure null default values apply - maybe standalone / portrait?

File: data/defaults.json [hunk 1]
```diff
@@ -31,7 +31,9 @@
     "silhouette": false,
     "version": 1.0,
     "config": null,
-    "logging": false
+    "logging": false,
+    "androidOrientation": null,
```
Comment (haydenbleasel) [inline data/defaults.json]: Default should be "portrait"

File: data/defaults.json [hunk 2]
```diff
@@ -31,7 +31,9 @@
...
+    "androidDisplay": null
```
Comment (haydenbleasel) [inline data/defaults.json]: Default should be "standalone"

File: readme.md [hunk 3]
```diff
@@ -54,7 +54,9 @@ To keep things organised, the configuration object contains 4 sub-objects: `file
         index: null,              // Path for the initial page on the site. `string`
         url: null,                // URL for your website. `string`
         silhouette: false,        // Turn the logo into a white silhouette for Windows 8. `boolean`
-        logging: false            // Print logs to console? `boolean`
+        logging: false,           // Print logs to console? `boolean`
+        androidOrientation: null, // Sets the orientation in manifest.json for Android Chrome `string`
```
Comment (haydenbleasel) [inline readme.md]: Again, default should be "standalone"

File: readme.md [hunk 4]
```diff
@@ -54,7 +54,9 @@ To keep things organised, the configuration object contains 4 sub-objects: `file
...
+        androidDisplay: null      // Sets the display value in manifest.json for Android Chrome `string`
```
Comment (haydenbleasel) [inline readme.md]: Again, default should be "standalone"

**Comment by haydenbleasel** [comment]
Hey @euangoddard, merging this issue into Favicons 4.0. I'd love for you to join in the discussion: #51

**Comment by euangoddard** [comment]
Will take a look

**Pull request closed without merging**